Bar chart changes (#212)
patternfly/training-scenarios: react-charts/bar-chart/intro.md

PatternFly React charts are based on the
[Victory chart](https://formidable.com/open-source/victory/docs/victory-chart/)
library, along with additional functionality, custom components, and theming for
PatternFly.

> PatternFly provides a collection of React-based components you can use to build

Reviewer: ```suggestion
PatternFly provides a collection of React-based components that can be used.
```

Reviewer 2: @gheake2 I really like this edit you did, simplifying it to end on "components." This reminds me of one piece of clarity I wanted to share with you before I forgot :D Per our Slack convo, replacing "we" with "you" or cutting "you" in instructions where we can is spot on! By cutting "you" in instructions, I'm referring to situations like changing "You can click this chart" to "Click this chart." We don't need to eliminate "you" everywhere, just for instruction pieces where we can. I like that you cut this sentence down; I think that's perfect. It just reminded me of what we were talking about with pronouns, so wanted to share here so that it didn't escape my mind!

> patterns with consistent markup, styling, and behavior.

Reviewer 3: why is this a new sentence, I think this might be a mistake

PatternFly React charts are composed using separate components, so features are

> more easily customizable. This course covers PatternFly bar chart - starting

Reviewer: ```suggestion
more easily customizable. This course covers PatternFly bar chart, starting
```

> PatternFly React charts are based on the [Victory chart](https://formidable.com/open-source/victory/docs/victory-chart/) library, along with additional functionality, custom components, and theming for PatternFly.

Author: updated from your PR
